README.md

```diff
@@ -50,16 +50,36 @@ Use for:
 
 ## Getting Started with Local Tooling
 
+### Download Skaffold binary 
+
+-  To download the latest Linux build, run:
+    -  `curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+-  To download the latest OSX build, run:
+    -  `curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-darwin-amd64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+
+### Build from source TL;DR
```

Comment:
Do you mind splitting this up into its own doc at `docs/contributing/building.md`?